### DISCUSSION TITLE
Include "internal price" in Time Tracking / All Times "edit"

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/Paid-plugins#102] [Questions & Answers]
The edit dialogue in Time Tracking / All times allows us to change the hourly rate and apply it retroactively to previous time entries in bulk, which is amazing.

Would it be possible to add the ability to change the "internal rates" as well? 

As a non-profit, we may have slightly different needs compared to "normal" companies.
We will use Kimai to report to the granting authority how we spend the funding for our projects. Often, the funding rate is 80% of the actual personnel costs.  We would like to use the "hourly rate" for the actual costs and track the funding rate as internal costs. When the hourly rate changes, in our case the internal rate also changes. The exact hourly rate can only be determined retroactively for a number of reasons. To keep our budget accurate, it would be amazing if we could change both rates. 

It would be even better if we could set the internal rate at a percentage of the hourly rate for selected projects.

**Comment by kevinpapst** [comment]
This repository is ONLY for posting topics that are related to paid plugins.
Please post here: https://github.com/kimai/kimai
